AnimatedWordCloud/Utils/Data/AllocationData.py

```diff
@@ -15,11 +15,15 @@ class AllocationInFrame:
     Positions and size of each words in a frame
     """
 
-    def __init__(self) -> None:
+    def __init__(self, from_static_allocation: bool) -> None:
```

Comment:
Why is "from_static_allocation" needed?

Reply:
Further implementation: make static frame's time longer